src/main/java/org/spdx/sbom/gradle/maven/PomResolver.java

```diff
@@ -173,16 +173,27 @@ public Map<String, PomInfo> effectivePoms(List<ResolvedArtifactResult> resolvedP
                       .map(
                           d ->
                               ImmutableDeveloperInfo.builder()
-                                  .name(Optional.ofNullable(d.getName()))
-                                  .email(Optional.ofNullable(d.getEmail()))
-                                  .organization(Optional.ofNullable(d.getOrganization()))
+                                  .name(presentIfNotEmpty(d.getName()))
+                                  .email(presentIfNotEmpty(d.getEmail()))
+                                  .organization(presentIfNotEmpty(d.getOrganization()))
                                   .build())
                       .collect(Collectors.toList()))
               .build());
     }
     return effectivePoms;
   }
 
+  private Optional<String> presentIfNotEmpty(String s) {
```

Comment:
this is personal preference: presentIfNotEmpty -> OptionalOfNonEmpty

Reply:
Updated